integrationtests/supervisor/integrationtest_supervisor.py

```diff
@@ -46,54 +54,9 @@ def assert_pipeline_exit_without_error(res: dict) -> None:
     assert exit_msg["exit_msg"]["exitcode"] == 0
 
 
-def test_mnist() -> None:
-    pipeline_config = load_config_from_file(MNIST_CONFIG_FILE)
-    print(pipeline_config)
-
-    res = parse_grpc_res(
-        supervisor.start_pipeline(
-            StartPipelineRequest(
-                pipeline_config=SupervisorJsonString(value=json.dumps(pipeline_config)),
-                start_replay_at=0,
-                maximum_triggers=1,
-            )
-        )
-    )
-    pipeline_id = res["pipeline_id"]
-
-    print(f"start pipeline: {res}")
-    assert pipeline_id >= 0
-
-    get_pipeline_status_res = parse_grpc_res(wait_for_pipeline(pipeline_id))
-    print(get_pipeline_status_res)
-    assert_pipeline_exit_without_error(get_pipeline_status_res)
-
-
-def test_one_experiment_pipeline() -> None:
-    pipeline_config = load_config_from_file(DUMMY_CONFIG_FILE)
```

Comment:
we have a func `test_two_experiment_pipelines` which covers entirely what this function covers.

So I remove this func to marginally save some integration test time.